.github/scripts/toolchain/build-mingw-crt.sh

```diff
@@ -14,7 +14,7 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$MINGW_BUILD_PATH/Makefile" ] ; then
 
     $SOURCE_PATH/$MINGW_VERSION/mingw-w64-crt/configure \
         --prefix=$TOOLCHAIN_PATH/$TARGET \
-        --build=x86_64-linux-gnu \
+        --build=$HOST \
         --host=$TARGET \
```

Comment:
In other places "--target=$TARGET" is used

Reply:
Yes, I've already fixed it in #115 but forget to update it here.

Reply:
For building the toolchain (binutils, GCC) the complete triplet ``build`/`host`/`target`` should be specified. For building runtime libraries, the `target` does not make sense and `host` == `target`.